dcpy/connectors/edm/recipes.py

```diff
@@ -1,15 +1,59 @@
+import argparse
+import cerberus
+import csv
 import json
+from pathlib import Path
+import shutil
+import sys
+from typing import Optional
+import yaml
 
 from sqlalchemy import text, update, Table, MetaData
 
+
 from dcpy import DCPY_ROOT_PATH
 from dcpy.utils import s3
+from dcpy.utils.logging import logger
 from dcpy.utils import postgres
+from dcpy.utils import versions
+
+# In order to keep things sane, maybe we should allow recipes to import
+# publishing but not the other way around?
+from . import publishing
```

Comment:
I like this for now, but maybe long-term we could move all `publishing` functions that are used here to something like `dcpy.utils.s3`? and same for anything `publishing` might import from `recipes`?

Reply:
I actually think (and @fvankrieken might also have thoughts) that there's a lot in `recipes` that isn't strictly about recipes, e.g. importing stuff into a database. I think a lot of this build stuff really belongs elsewhere. Maybe something like `dcpy.products` which would be responsible for the steps to build a product. Thoughts? @damonmcc

Reply:
true, or maybe `dcpy.builds`

Reply:
Yeah eventually that might make sense. For now I'm fine with it in `recipes` and keeping this file as related to everything starting a build, including the recipes datasets as well as grabbing build metadata